app/src/main/java/com/android/sample/resources/C.kt

```diff
@@ -147,7 +150,7 @@ object C {
     const val MEAL_DB_CATEGORY_ARRAY = "categories"
 
     // FirestoreRecipeRepository
-    const val FIRESTORE_COLLECTION_NAME = "recipes"
+    const val FIRESTORE_COLLECTION_NAME = "test"
```

Comment:
It looks like you may have missed removing the test collection name. It would be better to use the recipes collection from Firestore instead.
```suggestion
    const val FIRESTORE_COLLECTION_NAME = "recipes"
```

Reply:
Thanks, for the catch up i will switch it to "recipes with images" which is now the official database.

Reply:
Thanks for letting us know, @Ylanv! Switching to 'recipes with images' makes sense, and we appreciate the heads-up about the temporary black images on SwipePage, this helps us stay informed. Looking forward to the next PR!